src/integrationTest/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/newinitiate/PostInitiateByIdControllerTest.java

```diff
@@ -1374,5 +1374,77 @@ void should_not_trim_values_for_fields_that_are_excluded_during_task_initiation(
             );
 
     }
+
+    @Test
+    void should_throw_bad_request_when_client_specific_mandatory_fields_are_missing_during_task_initiation()
```

Comment:
Can we rename this as it's not "client specific" but rather initiation_manadatory_fields_required?